data/initrd/scripts/url/hmc/menu

```diff
@@ -0,0 +1 @@
+HMC
```

Comment:
>>> On 2/18/2016 at 04:25 AM, Steffen Winterfeldt <notifications@github.com> wrote:

> My impression was that HMC is a well-known term for s390 admins. But you're 
> from the s390 side. If you feel more comfortable with 'Hardware Management 
> Console', then so be it.

It is a well-known term, but I tend towards being more explanatory than not, when possible.  My comfort level was more in consideration of others that might not be as familiar.